### PR TITLE
Upgrade to Phoenix 1.7

### LINKED ISSUE #187: Phoenix update to version 1.7 🦅
Release notes: https://www.phoenixframework.org/blog/phoenix-1.7-final-released

Migration guide: https://gist.github.com/chrismccord/00a6ea2a96bc57df0cce526bd20af8a7

### PR DESCRIPTION
**Pull request by ruilopesm**
Closes #187.

**Disclaimer**:  This is not a full application upgrade to Phoenix 1.7. As you can see [here](https://www.phoenixframework.org/blog/phoenix-1.7-final-released), views are now built in a different way. So that, I've only migrated one of them to the new version (`file_view.ex`). I'm expecting other people to migrate the ones in fault, along with tests for them.

### CODE REVIEW

**Review by nelsonmestevao — APPROVED** [review]

File: lib/bokken_web.ex [hunk 1]
```diff
@@ -17,14 +17,34 @@ defmodule BokkenWeb do
...
+      end
+
     quote do
-      use Phoenix.Controller, namespace: BokkenWeb
+      unquote(result)
 
       import Plug.Conn
       import BokkenWeb.Gettext
       import Bokken.Guards
       alias BokkenWeb.Router.Helpers, as: Routes
```
Comment (nelsonmestevao) [inline lib/bokken_web.ex]: Maybe this should go to the 1.6 version only 💪

Comment (ruilopesm) [reply lib/bokken_web.ex]: Why?

Comment (nelsonmestevao) [reply lib/bokken_web.ex]: From now on, we should use the sigil p for routes 💪

And stop using Routes helper module

Comment (nelsonmestevao) [reply lib/bokken_web.ex]: https://hexdocs.pm/phoenix/routing.html#verified-routes

**Pull request merged**